### PR TITLE
[Merged by Bors] - Add x11-window-title option

### PR DESCRIPTION
**Pull request by graysonguarino**
Adds the x11-window-title option from Mir. Defaults to "Ubuntu Frame".

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ license: GPL-3.0
 compression: lzo
 package-repositories:
   - type: apt
-    ppa: mir-team/release
+    ppa: mir-team/dev
```
Comment (AlanGriffiths) [inline snap/snapcraft.yaml:11]: We don't want to change this.

File: snap/hooks/configure [hunk 2]
```diff
@@ -73,6 +73,7 @@ config_entry "console-provider"   "vt"                                  "Require
 config_entry "vt"                 "4"                                   "Virtual terminal to use for display (0 to use current)"
 config_entry "cursor"             "auto"                                "Mouse pointer to use (auto|none|software)"
 config_entry "env-hacks"          "MIR_MESA_KMS_DISABLE_MODESET_PROBE"  "As Frame HAS to run mesa-kms we can safely override the probe for KMS everywhere"
+config_entry "x11-window-title"   "Ubuntu Frame"                        "Default window title when run as Mir on X"
```
Comment (AlanGriffiths) [inline snap/hooks/configure:76]: The [original issue](https://github.com/MirServer/mir/issues/2219) said "Ubuntu Frame simulator", what is the rationale for changing it?

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
We definitely don't want to use ppa:mir-team/dev in production. And we need to confirm the title.

File: snap/hooks/configure (hunk 2 above)
Comment (graysonguarino) [reply snap/hooks/configure:76]: There is no rationale (other than possibly to match the name of the package). Ubuntu Frame Simulator (capital S preferred?) seems to make more sense in context.

Comment (graysonguarino) [reply snap/hooks/configure:76]: Possibly Ubuntu Frame (Simulated).

Comment (AlanGriffiths) [reply snap/hooks/configure:76]: PM is happy with "Ubuntu Frame"

**Review by AlanGriffiths — APPROVED** [review]
LGTM

bors merge

**Pull request closed without merging**